doc/api/events.md

```diff
@@ -1819,6 +1819,21 @@ added: v14.5.0
 
 This is not used in Node.js and is provided purely for completeness.
 
+#### `event.initEvent(type[, bubbles, cancelable])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 3 - Legacy
+
+* `type` {string}
+* `bubbles` {boolean}
+* `cancelable` {boolean}
+
+Redundant with event constructors and incapable of setting `composed`.
+This is not used in Node.js and is provided purely for completeness.
```

Comment:
I would expand this to also clearly state that the WHATWG spec considers it deprecated and that users likely shouldn't use it at all.

Reply:
Updated description of API according to your suggestion. PTAL. :)